source/_integrations/keenetic_ndms2.markdown

```diff
@@ -2,30 +2,42 @@
 title: Keenetic NDMS2 Routers
 description: Instructions on how to integrate Keenetic NDMS2 Routers into Home Assistant.
 ha_category:
+  - Hub
   - Presence Detection
 ha_release: 0.54
 ha_codeowners:
   - '@foxel'
 ha_domain: keenetic_ndms2
 ---
 
-This integration offers presence detection by examining devices connected to a [Keenetic](https://keenetic.net/)
-Router running NDMS2 firmware.
+This platform offers presence detection by examining devices connected to a [Keenetic](https://keenetic.net/)
+Router running NDMS firmware versions 2.05 and up.
 
-To use a Keenetic router in your installation, add the following to your `configuration.yaml` file:
+<div class='note warning'>
+
+You need to enable telnet on your router for this integration to work.
+
+</div>
+
+To use a Keenetic router in your installation, configure it on the Integrations page or add the following to your `configuration.yaml` file:
```

Comment:
Please provide instructions for the user how to add this to the UI (e.g., a click path and such).

Furthermore, I assume the YAML configuration is now deprecated and will be imported, so maybe just remove it?